string PublisherUUID
string PublisherFeatureUUID
string SubscriberUUID
string SubscriberFeatureUUID
float64 Coefficient
